spec/spec_helper.rb

```diff
@@ -1,19 +1,15 @@
 # frozen_string_literal: true
 
-SUPPORT_CODECOV = Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.4")
-
 require "bundler/setup"
 require "simplecov"
 require "simplecov-console"
-require "codecov" if SUPPORT_CODECOV
 require "webmock/rspec"
 require "pry"
 
 WebMock.disable_net_connect!
 
 Dir["./spec/support/**/*.rb"].sort.each { |f| require f }
 SimpleCov.formatter = SimpleCov::Formatter::Console
-SimpleCov.formatter = SimpleCov::Formatter::Codecov if SUPPORT_CODECOV
```

Comment:
why remove this line only?

Reply:
It's the line that configures the coverage generating tool (SimpleCov) to use the formatter that sends the report to CodeCov. Without this line, the previous line remains in effect: report current code coverage to STDOUT.

Reply:
Compare the console output end in [a test from this PR](https://app.circleci.com/pipelines/github/honeycombio/beeline-ruby/999/workflows/52f6a9f5-ebce-4842-92ef-884c7d8e66aa/jobs/71185) … 

![Screen Shot 2021-09-14 at 6 26 27 PM](https://user-images.githubusercontent.com/517302/133342134-889a45e4-b384-4734-9d0a-032db9e8429d.png)


 to [a test from the past](https://app.circleci.com/pipelines/github/honeycombio/beeline-ruby/995/workflows/717907b7-499c-415f-9da9-ae267c83215a/jobs/70833).

![Screen Shot 2021-09-14 at 6 27 12 PM](https://user-images.githubusercontent.com/517302/133342145-b87a7455-3785-4fb7-921f-3e4a487a5056.png)